test/fops/file.c

```diff
@@ -97,6 +112,10 @@ obj_bulk_unique(int force)
 		if (ret != EBUSY)
 			testutil_die(ret, "session.drop: %s", new_uri);
 
+	if (explicit_transaction)
+		if ((ret = session->commit_transaction(session, NULL)) != 0 &&
+		    ret != EINVAL)
+			testutil_die(ret, "session.commit bulk unique");
```

Comment:
These two if-statements (and all the similar ones below) should be combined with an && since there is no else clause on either of them.